spec/models/local_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+describe Local do
+
+	context "When local's address is empty" do
+		it do
+			user = FactoryGirl.create(:user, name: "Junior1", email: "junior@hotmail.com")
+
+			local = Local.new
+			local.address = "Rua do Compositor"
+			local.user_id = user.id
+
+			expect(local.valid?).to be_truthy
+		end
+	end
+
+	context "When verifying local's user" do
+		user = FactoryGirl.create(:user, name: "Junior2", email: "junior@hotmail.com")
+
+		it "Should have an user" do
```

Comment:
You could write a spec for when user is set to nil. If you do so, you can wrap this it in a new context `when user is present` and remove the description for this it.

It would be something like this:

```
context 'When verifying local's user'
   context 'when user is present'
       it {  write your spec }
   end

  context 'when user is not present'
     it { write your spec }
   end
end
```